drfc_manager/pipelines/viewer.py

```
@@ -267,76 +330,101 @@
 
     process = None
-    stdout_file = None
-    stderr_file = None
     try:
-        stdout_file = open(proxy_stdout_log, 'w')
-        stderr_file = open(proxy_stderr_log, 'w')
         logger.info(f"Starting proxy server process: {' '.join(cmd)}")
+        env = get_subprocess_env(env_vars)
+        env["DR_VIEWER_CONTAINERS"] = json.dumps(containers)
+        logger.info(f"Environment variables for proxy process: {env}") 
+        log_file = open(log_file_name, "w")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)